guides/micronaut-docker-image/micronaut-docker-image.adoc

```diff
@@ -0,0 +1,404 @@
+common:header.adoc[]
+
+common:requirements.adoc[]
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+:exclude-for-build:maven
+
+== Building a Docker Image with the Micronaut Gradle Plugin
+
+The https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/[Micronaut Gradle Plugin] offers the following tasks to build Docker images:
+
+* `dockerBuild` - Builds a Docker Image using the Docker Gradle plugin
+* `dockerBuildNative` - Builds a Native Docker Image using GraalVM Native Image
+* `optimizedDockerBuild` - Build an optimized application and package it into a Docker image.
+* `optimizedDockerBuildNative` - Build a native image containing the optimized application and pakcage it into a Docker Image
+
+:exclude-for-build:
+
+:exclude-for-build:gradle
+
+== Building a Docker Image with the Micronaut Maven Plugin
+
+https://micronaut-projects.github.io/micronaut-maven-plugin/latest/[Micronaut Maven Plugin] offers several options to package your application as a Docker Image.
+
+* `docker` builds a Docker image with the application artifacts (compiled classes, resources, dependencies, etc).
+* `docker-native`: builds a Docker image with a GraalVM native image inside.
+* `docker-crac`: build a a pre-warmed, checkpointed Docker Image
+
+:exclude-for-build:
+
+== Sample App
+
+Add a controller which responds with the JSON payload in the apex route.
```

Comment:
apex route?

Reply:
the "/"

Reply:
https://github.com/micronaut-projects/micronaut-guides/pull/1239/commits/ae1d15bc1c2ffdbcc46a029ea728627acd24f3e3